packages/app/package.json

```diff
@@ -205,6 +205,7 @@
     "shortid": "^2.2.8",
     "store": "^2.0.12",
     "string-replace-loader": "^2.1.1",
+    "strip-json-comments": "^2.0.1",
```

Comment:
I don't think we need this as we already use JSON5 to parse (https://json5.org/). The reason I did that RegEx was because I wanted the default colors which are commented out to be used as well, as we don't have those default colors built in.

So if you have:

```
//"editorActiveLineNumber.foreground": "#0b216f",
```

I want it to become:

```
"editorActiveLineNumber.foreground": "#0b216f",
```

But only for those kind of comments, not for comments that explain something. 

Maybe it works with these colors commented though, is worth testing.

Reply:
Yeah, seems like I completely misunderstood what that code did. 😃

Still, shouldn't that be
```js
replace(/^\s*\/\/"/gm, '"')
```
or is JSON5 resilient in missing quotes in property names?

Reply:
You're right! That's what it should be, big mistake on my side 😅.